jwst/resample/resample.py

```diff
@@ -364,9 +366,15 @@ def resample_many_to_one(self):
         self.update_exposure_times(output_model)
         self.output_models.append(output_model)
 
+        for img in self.input_models:
+            try:
+                del img.meta.iscale
+            except AttributeError:
+                pass
```

Comment:
This is not as critical as the other `try-except` block but I feel we should not have issues with deleting this attribute. So I would suggest removing `try-except`. It would be strange/unexpected (and maybe reveal some lack in our understanding of how things work here) if this would crash.

Reply:
Done. Thank you for the comments.